main.nf

```diff
@@ -42,6 +42,7 @@ def helpMessage() {
       --num_hits                    Number of peptide hits per spectrum (PSMs) in output file (default: '1')
       --fixed_mods                  Fixed modifications ('Carbamidomethyl (C)', see OpenMS modifications)
       --variable_mods               Variable modifications ('Oxidation (M)', see OpenMS modifications)
+      --phospho_rescoring           Phosphosite rescoring using the luciphor algorithm
```

Comment:
to be more general this could be re-written to: --localize_modifications (which expects a list of modifications that is then passed to luciphor if not empty).

Reply:
this could be great because the new version supports other mods as well